### DISCUSSION TITLE
Need Help with informer model

### DISCUSSION THREAD

**Discussion by Shashwat26Bhatnagar** [community/community#136071] [Programming Help]
### Select Topic Area

Question

### Body

I would appreciate your help in my ongoing project where I am using Informer.

**Comment by ItsXcr9** [comment]
explain more about it

**Comment by LiteBrite82** [comment]
Thanks for posting in the GitHub Community, @Shashwat26Bhatnagar ! 

We’ve moved your post to our Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.
